packages/pacman/build.sh

```diff
@@ -34,12 +37,13 @@ termux_step_post_configure() {
 termux_step_post_make_install() {
 	mkdir -p $TERMUX_PREFIX/etc/pacman.d
 	install -m644 $TERMUX_PKG_BUILDER_DIR/serverlist $TERMUX_PREFIX/etc/pacman.d/serverlist
+	install -m755 $TERMUX_PKG_SRCDIR/pacman-switch-main/pacman-switch.sh $TERMUX_PREFIX/bin/pacman-switch
 }
 
-termux_step_create_debscripts() {
-	echo "#!$TERMUX_PREFIX/bin/bash" > postinst
-	echo "mkdir -p $TERMUX_PREFIX/var/lib/pacman/sync" >> postinst
-	echo "mkdir -p $TERMUX_PREFIX/var/lib/pacman/local" >> postinst
-	echo "mkdir -p $TERMUX_PREFIX/var/cache/pacman/pkg" >> postinst
-	chmod 755 postinst
+termux_step_post_massage() {
+	mkdir -p ./var/lib/pacman/sync
```

Comment:
I think it may be better to create a `.placeholder` file in these folders in `post_make_install`.